loop-server/manifest.json

```diff
@@ -2,16 +2,30 @@
   "envs" : {
     "stage": {
       "test_types": [ "stack-check", "loadtest" ],
+      "stack_label": "loopsvrstg",
```

Comment:
Is this our label, or some existing OPs thing?
Above we used `"stack_label": "loopclientstage"`, which makes this `"loopsvrstg"` look inconsistent.